### DISCUSSION TITLE
Angular Three Examples

### DISCUSSION THREAD

**Discussion by IRobot1** [nartc/angular-three#47] [Show and tell]
Link for [Basic Starter Project](https://github.com/IRobot1/ng3-template).  See readme for starting from scratch.

I also started a project to convert [three.js VR examples](https://threejs.org/examples/?q=vr) to [Angular Three Examples](https://github.com/IRobot1/angular-three-examples)

Wiki contains code walk through, challenges and open problems that will need to be submitted here.

My plan is to try converting the VR hand input examples now that the Quest supports hand tracking.

Next, I'd like to convert all examples in order to validate ngt functionality.

Long term, I plan to use this in a business app for visualizing asset relationships in VR, creating a material library for VR and for creating an online VR Alchemy/Crafting game.

**Comment by Polyterative** [comment]
that was really helpful. We need a basic starter. I salvaged one frome your examples. Thanks
